snippet-extractor-output/snippets.yaml

```diff
@@ -3144,17 +3144,17 @@ Word.Paragraph.insertText:
 Word.ContentControl.tag:
   - >
     // Traverses each paragraph of the document and wraps a content control on
-    each with either a even or odd tags
+    each with either a even or odd tags.
 
     await Word.run(async (context) => {
         let paragraphs = context.document.body.paragraphs;
-        paragraphs.load("$none"); //Don't need any properties, we'll just wrap each paragraph with a content control
+        paragraphs.load("$none"); // Don't need any properties; we'll just wrap each paragraph with a content control.
 
         await context.sync();
 
         for (let i = 0; i < paragraphs.items.length; i++) {
             let contentControl = paragraphs.items[i].insertContentControl();
-            // For even we tag "even"
+            // For even we tag "even".
```

Comment:
```suggestion
            // For even, we tag "even".
```

Reply:
This file is auto-genned so I've made the update in the snippet itself.